modules/ROOT/pages/migrate-cl.adoc

```diff
@@ -1,49 +1,51 @@
 
 :toc:
 
 == Introduction
 
-To migrate from CL to FCOS, you must convert your old Container Linux configs, Ignition configs, and cloud-configs to a xref:fcct-config.adoc[Fedora CoreOS Config (FCC)] and adapt the contents for FCOS. Since many of the configuration options have changed, you should reference this page and https://github.com/coreos/fedora-coreos-tracker/issues/159[CL migration documentation] on GitHub.
+To migrate from CL to FCOS, you must convert your old Container Linux Configs, Ignition configs, or `cloud-config` files to a xref:fcct-config.adoc[Fedora CoreOS Config (FCC)] and adapt the contents for FCOS. Since many of the configuration details have changed, you should reference this page and https://github.com/coreos/fedora-coreos-tracker/issues/159[CL migration documentation] on GitHub.
 
 == Installation changes
-* To accommodate changes to the Ignition specification, the `coreos-install` script has been deprecated in favor of https://github.com/coreos/coreos-installer[`coreos-installer`].
-* The autologin kernel command-line parameter is not currently supported in FCOS. To preview FCOS, you can use xref:bare-metal.adoc[Live PXE].
-* Certain CL image types are not yet supported in FCOS. Refer to the https://getfedora.org/coreos/download/[Download page] to see a list of the available image types.
-* There are no official Vagrant cloud images for FCOS, and the team has no plans to create OCI container images for FCOS. FCOS is intended to be a host OS for running containers, rather than a host for producing containers.
+
+The following changes have been made to the installation process:
+
+* The `coreos-install` script has been deprecated in favor of https://github.com/coreos/coreos-installer[`coreos-installer`], which offers similar functionality.
+* The `autologin` kernel command-line parameter is not currently supported in FCOS.
+* Certain CL platforms, such as Vagrant, are not yet supported in FCOS. Refer to the https://getfedora.org/coreos/download/[Download page] to see a list of the available image types.
 
 == Configuration changes
-The xref:fcct-config.adoc[Ignition specification] has changed in both syntax and function. As a result, consider the following when you write new FCC files:
+
+When writing FCC files, keep the following changes in mind:
 
 * Users and groups are defined in the xref:ign-passwd.adoc[`passwd`] node of the FCC file.
-//* CLC --> FCC changes. Need more info.
-//* coreos-metadata rename - need more info.
-//* PXE custom OEMs - need more info.
+* `coreos-metadata` is now https://github.com/coreos/afterburn/blob/master/README.md[Afterburn]. You will have to evaluate the environment variables in your systemd unit files. For the sake of comparison, you can refer to the https://github.com/coreos/afterburn/blob/master/docs/container-linux-legacy.md[legacy documentation].
+* PXE images can be customized with a bundled Ignition config. Follow the xref:bare-metal.adoc[PXE live boot] instructions.
```

Comment:
This isn't actually documented in that file, and I don't see any PRs adding such docs either.  We should have a bullet point here with the info described in my previous comment.

Reply:
It was a poor attempt to point to the example of using config.ign to boot a PXE. I will fix it.

Reply:
I misunderstood this whole thing, but now I see where I went wrong. I will fix this in my next commit.